### PR TITLE
Made examples Readme.md more readable

### PR DESCRIPTION
**Pull request by ReblochonMasque**
Committed minor alterations of the examples Readme.md file in order to make contact and introduce myself.

### CODE REVIEW

File: examples/Readme.md [hunk 1]
```diff
@@ -17,10 +20,13 @@ Simple cellular automata of a fire spreading through a forest of cells on a grid
 ### PD_Grid
 Grid-based demographic prisoner's dilemma model, demonstrating how simple mimicing can lead to the emergence of widespread cooperation -- and how a model activation regime can change its outcome.
 
-### Schelling.py (needs Readme)
+### Schelling.py 
+(needs Readme)
 Simple implementation of the classic Schelling segregation model.
 
-### Tutorial Bollzmann Wealth Model (needs Readme)
+### Tutorial Bollzmann Wealth Model
```
Comment (dmasad) [inline examples/Readme.md]: Should be spelled 'Bolzmann' -- otherwise, looks good!

Comment (ReblochonMasque) [inline examples/Readme.md]: indeed.

**Comment by ReblochonMasque** [comment]
Corrected typo

**Comment by jackiekazil** [comment]
Is this ready for review?

**Comment by ReblochonMasque** [comment]
Yes.

**Comment by jackiekazil** [comment]
Great... i will try to look through this today.... i am on a train for 10+ hours, so I think I will have enough time. :-D

File: examples/ConwaysGameOfLife/Readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Conway's Game Of Life
+
+[The Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life), also known simply as Life, is a cellular automaton devised by the British mathematician John Horton Conway in 1970.
+
+The "game" is a zero-player game, meaning that its evolution is determined by its initial state, requiring no further input. One interacts with the Game of Life by creating an initial configuration and observing how it evolves, or, for advanced "players", by creating patterns with particular properties.
```
Comment (jackiekazil) [inline examples/ConwaysGameOfLife/Readme.md]: maybe... 'requiring no further input by a human'. <--- I feel like it needs a little more clarification. Yes? No?

File: examples/ConwaysGameOfLife/Readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## Conway's Game Of Life
+
+[The Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life), also known simply as Life, is a cellular automaton devised by the British mathematician John Horton Conway in 1970.
```
Comment (jackiekazil) [inline examples/ConwaysGameOfLife/Readme.md]: Should Life be in quotes or italicized? (I think I am being picky. Feel free to push back.)

Comment (jackiekazil) [inline examples/ConwaysGameOfLife/Readme.md]: Should Life be in quotes or italicized? (I think I am being picky. Feel free to push back.)

**Comment by jackiekazil** [comment]
@ReblochonMasque These are great updates. I made two minor suggestions. When those are complete,  I will merge into master.

**Comment by ReblochonMasque** [comment]
Thank you for reviewing and your nice comment Jacquie. Your suggestions have been added.
Fred

**Comment by jackiekazil** [comment]
Looks good. I am going to merge these in.

**Pull request merged**